java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java

```diff
@@ -114,7 +113,7 @@ private Path searchForArtifactInDir(Path directory) throws ArtifactNotFoundExcep
     Files.list(directory)
         // filter out files that don't end in .war or .jar
         .filter((path) -> {
-          String extension = FileUtil.getFileExtension(path);
+          String extension = com.google.common.io.Files.getFileExtension(path.toString());
```

Comment:
Unfortunately no, fully qualified reference is required here since we use `java.nio.file.Files` elsewhere in this class.